src/legacy/core_plugins/interpreter/public/lib/batched_fetch.ts

```diff
@@ -47,19 +70,19 @@ export function batchedFetch({ ajaxStream, serialize, ms = 10 }) {
     reset();
   };
 
-  return ({ functionName, context, args }) => {
+  return ({ functionName, context, args }: any) => {
     if (!timeout) {
       timeout = setTimeout(runBatch, ms);
     }
 
-    const request = {
+    const request: Request = {
       functionName,
       args,
       context: serialize(context),
     };
 
     // Check to see if this is a duplicate server function.
-    const duplicate = Object.values(batch).find(batchedRequest =>
+    const duplicate: any = Object.values(batch).find((batchedRequest: any) =>
```

Comment:
What's the purpose of declaring `duplicate` and `batchedRequest` as `any`? Won't the Typescript inference work here?

Reply:
I believe it did not infer, otherwise I would not add those here. Will check this line in next PR.